### DISCUSSION TITLE
Copilot purchase issue

### DISCUSSION THREAD

**Discussion by ltyec** [community/community#81529] [Copilot Conversations]
### Select Topic Area

Bug

### Body

After submitting to the purchase, it really direct to the page that shows `GitHub Copilot is now ready to pair!`. But it is not actually purchased and I had no access to copilot. It happend when my trial goes to the end and I don't know how to figure it out

**Comment by rbroidaguard1** [comment]
It's happening to me too. I called the credit card company, and they said GitHub is sending them an incorrect card expiration date. I entered the correct expiration date on GitHub's page, but that's not what they sent to the card company. I've opened a support ticket.

**Reply by Oddly** [reply]
Same issue here. Github sponsors works, Github Copilot doesn't. Opened a support ticket as well.

**Reply by rbroidaguard1** [reply]
GitHub replied to my support ticket, and the person who replied doesn't know how to fix it:

--------------

... Unfortunately, we don't get much information from the credit card company when a credit card transaction is declined. In this case, here's the error message we are receiving:
 
402 - [card_error/card_declined/do_not_honor] Your card was declined.
 
Could you try liaising with your bank again to confirm, as they should be able to tell you what's happening, then updating the payment method (even if you will be using the same details) should trigger a fresh attempt from your side. ... As an alternative, many of our customers utilize Paypal when their bank blocks GitHub transactions. Perhaps this would be a suitable alternative for you if your bank is unable to assist.

--------------
 
In my ticket I already told them the reason the bank gave for why they rejected the card: because GitHub sent the wrong expiration date. I had entered the correct expiration date on the GitHub site, but GitHub sent an incorrect date to the bank.

Deleting and re-entering the card information doesn't help, and I don't have a PayPal account. 

Hopefully someone at GitHub will do something about this issue, i.e. fix the bug.
